bin/docserv-stitch

```diff
@@ -174,7 +177,13 @@ for file in *.xml; do
         "$input_dir/$file: Some lang attributes are not supported by your configuration INI. Check for occurrences of the following unsupported lang attribute(s):\n$unrecognized_languages"
   fi
 
-
+  # Iterate over all XSLT stylesheets (starting with "check") and
+  # apply them to the configuration XML file. Check result of xsltproc.
+  # Each XSLT stylesheet prints "0" (=success) or "10" (=failure).
+  for xslt in $checks_dir/check-*.xsl; do
+    result=$(xsltproc $xslt $file)
+    [ 0 -ne $result ] && out "Check failed for $xslt."
```

Comment:
Hm, if you look at the other checks, I generally avoided just going out(), instead I am validating everything and showing the list of all issues in all files at the end. While this tends to make longer the list of issues displayed at once, it avoids the frustrating experience where validation fails, you fix the issue, and then validation fails again with a different issue.

Reply:
Yes, I know, I fully agree. I had the same idea, but I couldn't finish it today. So I've published what I have and will fine-tune it afterwards. :smiley: